src/extlibs/enlsvg/ENLSVGAlgorithm.cpp

```diff
@@ -0,0 +1,457 @@
+<<<<<<< HEAD:src/extlibs/enlsvg/ENLSVGAlgorithm.cpp
+#include "extlibs/enlsvg/ENLSVGAlgorithm.h"
+
+#include <algorithm>
+
+#include "extlibs/enlsvg/ENLSVGGraph.h"
+#include "extlibs/enlsvg/Grid.h"
+#include "extlibs/enlsvg/IndirectHeap.h"
+#include "extlibs/enlsvg/LineOfSightScanner.h"
+=======
+#include "extlibs/enlsvg/Pathfinding/enlsvg_algorithm.h"
+
+#include <algorithm>
+
+#include "extlibs/enlsvg/Pathfinding/enlsvg_graph.h"
+#include "extlibs/enlsvg/Pathfinding/grid.h"
+#include "extlibs/enlsvg/Pathfinding/indirect_heap.h"
+#include "extlibs/enlsvg/Pathfinding/line_of_sight_scanner.h"
+>>>>>>> 2330f76f64c53b1a23c58fb60b72d47482ba3da5:src/extlibs/enlsvg/enlsvg_algorithm.cpp
```

Comment:
you have git conflict markers, which means that you have some conflicts that haven't been resolved yet